### ISSUE TITLE
Move naming and tagging to use STIG ID (RHEL-06-XXXXX) instead of Vuln ID (V-XXXXX)

### ISSUE THREAD

**Issue by shepdelacreme** [MindPointGroup/RHEL6-STIG#153]
Should move tagging and task naming to use the STIG ID form to be consistent with the RHEL7 role.

**Comment by jamescassell** [comment]
would also be good to have a var for each item so you can en/disable items w/ host_vars

**Comment by shepdelacreme** [comment]
@jamescassell yes that is documented here: https://github.com/MindPointGroup/RHEL6-STIG/issues/62

**Comment by LeamHall** [comment]
What will the process be when the STIG data changes drastically? We're seeing this in some of the RHEL 7 newer STIGs. Also, I've been told but have not personally confirmed that DISA is re-using VIDs across STIGs for totally different items.

**Comment by shepdelacreme** [comment]
@LeamHall yeah we experienced the "big change" with the recent V2 switch for the RHEL 7 stigs. Unfortunately not much we can do but realign our stuff.

For the VID vs STIG ID thing...if they are reusing VIDs then this is more of a reason to switch to use STIG ID for the identifier since that won't be reused.
